miasm2/analysis/depgraph.py

```diff
@@ -392,6 +423,20 @@ def __init__(self, ira):
 
         # The IRA graph must be computed
         self._ira.gen_graph()
+        self.cb_follow = [follow_simp_expr, follow_mem_cst]
```

Comment:
As `cb_follow` seems to be class dependent and not instance dependent, you could use a class attribute instead